### PR TITLE
[maintenace] updates to work with ember-cli-fastboot@2 and fastboot@2

### PR DESCRIPTION
**Pull request by gabrielcsapo**
Everything works, when moving to ember-cli-document-title-northm.

### CODE REVIEW

**Comment by gabrielcsapo** [comment]
@ef4 would you be able to look at this when you get the chance? We are working on getting prember working with ember-cli-addon-docs

**Comment by scalvert** [comment]
@ef4 to give you some context, we've forked `ember-cli-document-title` as it's basically unmaintained. We've attempted to contact the owner, but no response. There's 11 open PRs. 

As such we're trying to ensure critical fixes are still pushed ahead, with a slightly longer term plan of moving it to its own org (`ember-cli-document-title/ember-cli-document-title`). This PR will at least move us onto a newer version of that addon that works.

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```
Comment (scalvert) [inline .npmrc]: Do we need this file?

File: index.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+const BroccoliDebug = require('broccoli-debug');
+const BroccoliMergeTrees = require('broccoli-merge-trees');
+
+const Prerender = require('./lib/prerender');
```
Comment (ef4) [inline index.js]: It was intentional that we `require`'d these things lazily on demand.

By requiring them up front, people running things like `ember help` or `ember generate` are forced to parse and evaluate all those dependencies even though they won't use them.

File: lib/prerender.js [hunk 3]
```diff
@@ -77,13 +79,11 @@ class Prerender extends Plugin {
...
 
-
-
-
     await writeFile(path.join(this.outputPath, 'package.json'), JSON.stringify(pkg));
 
     let app = new FastBoot({
-      distPath: this.inputPaths[0]
+      distPath: this.inputPaths[0],
+      resilient: true,
```
Comment (ef4) [inline lib/prerender.js]: I think it was also intentional that we did not set `resilient` here. We want the build to fail if some of the prerendered pages can't actually prerender.

**Comment by ef4** [comment]
Thanks for working on this, the work got rolled into #60.

**Pull request closed without merging**